markdown/usage/nf_core_tutorial.md

```diff
@@ -194,15 +265,26 @@ To help with this, the `nf-core download` command automates the fetching of requ
 * Download the _nf-core/rnaseq_ pipeline for offline use using the `nf-core download` command
 
 ## Creating nf-core pipelines
+
 ### Using the nf-core template
-The heart of _nf-core_ is the standardisation of pipeline code structure. To achieve this, all pipelines adhere to a generalised pipeline template. The best way to build an _nf-core_ pipeline is to start by using this template via the `nf-core create` command. This launches an interactive prompt on the command line which asks for things such as pipeline name, a short description and the author's name. These values are then propagated throughout the template files automatically.
+
+The heart of _nf-core_ is the standardisation of pipeline code structure.
+To achieve this, all pipelines adhere to a generalised pipeline template.
+The best way to build an _nf-core_ pipeline is to start by using this template via the `nf-core create` command.
+This launches an interactive prompt on the command line which asks for things such as pipeline name, a short description and the author's name.
+These values are then propagated throughout the template files automatically.
 
 ### TODO statements
-Not everything can be completed with a template and all new pipelines will need to edit and add to the resulting pipeline files in a similar set of locations. To make it easier to find these, the _nf-core_ template files have numerous comment lines beginning with `TODO nf-core:`, followed by a description of what should be changed or added. These comment lines can be deleted once the required change has been made.
 
-Most code editors have tools to automatically discover such `TODO` lines and the `nf-core lint` command will flag these. This makes it simple to systematically work through the new pipeline, editing all files where required.
+Not everything can be completed with a template and all new pipelines will need to edit and add to the resulting pipeline files in a similar set of locations.
+To make it easier to find these, the _nf-core_ template files have numerous comment lines beginning with `TODO nf-core:`, followed by a description of what should be changed or added.
+These comment lines can be deleted once the required change has been made.
+
+Most code editors have tools to automatically discover such `TODO` lines and the `nf-core lint` command will flag these.
+This makes it simple to systematically work through the new pipeline, editing all files where required.
 
 ### How nf-core software packaging works
+
 The only hard requirement for all _nf-core_ pipelines is that software must be available in Docker images. However, it is recommended that pipelines use the following methodology where possible:
 
 1. Software requirements are defined for Conda in `environment.yml`
```

Comment:
Should we specify here that the packages should be available in the conda-forge / bioconda / etc. channels, as we had the case in the past that someone provided their own channel?

Reply:
I guess it will still work if people use their own channel in the `environment.yml` file? I think that's probably a little too much detail for the tutorial though to be honest..